pandas/tests/test_resample.py

```diff
@@ -13,6 +13,7 @@
 import pandas as pd
 import pandas.tseries.offsets as offsets
 import pandas.util.testing as tm
+from pandas.util._test_decorators import skip_if_no_mpl
```

Comment:
can you write these as
``import pandas.util._test_decorators as td``

then use ``td.skip_if_no_mpl``

I think a bit simpler